playbooks/documentation.md

```diff
@@ -0,0 +1,78 @@
+---
+title: Documentation
+description: How and where to document or share content
+---
+
+# Documentation
+
+As engineers we operate in many parallel forums: GitHub, Jira, Slack, Google docs, e-mail, Notion, and so on. What
+content should go where?
+
+## Documentation or discussion of a specific system
+
+When documentation relates to a particular system, leverage in-repository `README`s or `doc/`s wherever possible.
+This includes when the consumers may not exclusively be engineers.
+
+E.g.:
+
+- How to set-up a local development environment
+- Definitions of domain models
+- API documentation
+
+It _almost_ goes without saying, but other repo-specific interactions such as presenting the rationale for a design
+or offering code-review should leverage the built-in mechanisms of
+[commit messages](/playbooks/engineer-workflow.md#commits),
+[pull requests](/playbooks/engineer-workflow.md#pull-requests), and comments.
+
+## Public versus private
+
+When creating a repo, contributing a doc, or discussing in-progress work, consider whether it's appropriate for
+this content to be public or private.
+
+Content that should always be private:
+
+- Contact or role details for individuals
+- Information about candidates or new hires before they've joined or consented to announce their move
+- Production data or logs (including indirectly through migration output, etc.)
+- Repositories representing substantial business value, novelty, or criticality (such as the main API, content
+  management system, or auction operator tools)
+
+Content that _maybe_ should be private, depending on its sensitivity:
+
+- Security or compliance topics, including pending security work items or vulnerability-handling procedures
+- Incident/support instructions, including data recovery or incident-mitigation procedures
+- Sensitive business rules, definitions, or implementations (tiering, targeting, buyer/bidder qualification, spam
+  heuristics, search ranking algorithms, data partnerships...)
+- Un-announced feature work
+- Industry or user research or analysis that may hold some competitive value
+- References to private content (according to the criteria above) including documentation of private systems, links
+  to private gists that may include private data or logs, etc.
+
+## General engineering content
+
+This public [artsy/README](https://github.com/artsy/readme) repo is our default location for shared engineering
+content such as:
+
+- onboarding resources
+- team procedures and playbooks
+
+The private [artsy/potential](https://github.com/artsy/potential) repo supplements that with internal resources
```

Comment:
“That” is a bit unclear in this context, I’d recommend replacing with “artsy/README” instead.

Also, I’d s/internal/private so that we stay consistent with terminology.